google-cloud-graalvm-samples/graalvm-samples-client-library/spanner-sample/src/main/java/com/example/SpannerSampleApplication.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.InstanceAdminClient;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+
+public class SpannerSampleApplication {
+
+  private static final String TEST_INSTANCE_ID = "graalvm-sample-test-instance";
+  private static final String TEST_DATABASE_ID = "graalvm-sample-test-database";
+
+  /**
+   * Runs the Spanner sample application.
+   *
+   * <p>This application should be run with the Spanner emulator for testing purposes.
+   */
+  public static void main(String[] args) {
+    System.out.println("Running the Spanner Sample.");
+
+    SpannerOptions options = SpannerOptions.newBuilder()
+        .setProjectId("DUMMY_PROJECT_ID")
```

Comment:
Is this a dummy project because the emulator does not use it? Consider leaving a comment.

Reply:
Done. Removed - Yeah I just set it initially for the emulator but realized it is not necessary.